doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+
+!media lfr/XS_Hom_Vs_Reg.jpg
+       style=width:50%
+       id=XS_Hom_Vs_Reg
+       caption=Relative difference of assembly average cross section to hexagonal ring-wise fuel cross section for U-238 capture
+
+For the non-fuel region, the setting is almost similar to the first step input except `c_externalspectrum_ufg=rzmflx` instead of `l_twodant=T` in the `control` block and `i_externalspectrum(composition #)=TWODANT zone #` in the `material` block.
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/NonFuel/LFR_127Pin_NonFuel_9g.mcc3.sh
+
+### Run scripts and merging ISOTXS files
+
+The process to prepare the final isoxml file is described here. First, the TWODANT running script is run as
+
+```
+cd Step1
+./LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh
+```
+
+Once the `LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh.rzmflx` file is created, two scripts need to be run as
+
+```
+cd ../Step2/Fuel
+./LFR_127Pin_Fuel_1D_9g.mcc3.sh
+cd ../NonFuel
+./LFR_127Pin_NonFuel_9g.mcc3.sh
+cd ../../
+```
+
+The order to run two scripts doesn't matter. Once the above two scripts are run, `LFR_127Pin_Fuel_1D_9g.mcc3.sh.ISOTXS` and `LFR_127Pin_NonFuel_9g.mcc3.sh.ISOTXS` files are created. The next step is to convert those two ISOTXS files to isoxml files by running
+
+```
+isoxml-opt -n ./Step2/Fuel/LFR_127Pin_Fuel_1D_9g.mcc3.sh.ISOTXS
+isoxml-opt -n ./Step2/NonFuel/LFR_127Pin_NonFuel_9g.mcc3.sh.ISOTXS
+```
+
+, which creates `LFR_127Pin_Fuel_1D_9g.mcc3.sh.xml` and `LFR_127Pin_NonFuel_9g.mcc3.sh.xml` files in respective folders. Before merging these two files, the library ID of `LFR_127Pin_NonFuel_9g.mcc3.sh.xml` needs to be manually modified from $1$ to $2$ (`Multigroup_Cross_Section_Library ID="2"`) as
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step2/NonFuel/LFR_127Pin_NonFuel_9g.mcc3.sh.xml start=<Multigroup_Cross_Section_Library end=<Tabulation> include-end=False
+
+Without this manual change of the library ID, two xml files cannot be merged. Finally, two xml files are merged using the merging capability of ISOXML to produce the final xml file as
```

Comment:
Is there a way to populate the ID in ISOXML properly? Like if ISOTXS stores an ID, that ID can be used by the converter, or we can add a command-line option like `--isoxml-isotxs-id` to assign the ID when converting. This does not have to be in place here.

Reply:
Unfortunately, there are limitations to populate the ID in ISOXML as intended. Regarding this, I made a major change in the MC2-3 script and the Griffin input as well as the library. I added detailed description on ISOTXS->ISOXML conversion part and isoxml merging process.